README.md

```diff
@@ -186,6 +186,8 @@ extension has the following configuration options:
   internal diagnostic information.
 - `deno.lint`: Controls if linting information will be provided by the Deno
   Language Server. _boolean, default `true`_
+- `deno.maxTsServerMemory`: Maximum amount of memory the TypeScript isolate can
```

Comment:
The vscode setting name for typescript is `typescript.tsserver.maxTsServerMemory`